pkgs/top-level/all-packages.nix

```
@@ -10229,6 +10229,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation IOKit;
     protobuf = protobuf3_21;
   };
+  netdataCloud = netdata.override {
+    withCloud = true;
```

The conditional should go *here*:
```suggestion
    withCloud = !stdenv.isDarwin;
```

---

Or you should guard netdataCloud behind an assert.

---

Thanks, fixed.